packages/terra-dropdown-button/src/_DropdownList.jsx

```diff
@@ -160,11 +179,31 @@ class DropdownList extends React.Component {
    * @return {Array<React.ReactNode>} the array of children
    */
   cloneChildren() {
-    return React.Children.map(this.props.children, (child, index) => React.cloneElement(child, {
-      isActive: index === this.state.active,
-      requestClose: this.props.requestClose,
-      'data-terra-dropdown-list-item': true,
-    }));
+    return React.Children.map(this.props.children, (child, index) => {
+      const currentItemLabel = this.props.children[index]?.props.label;
+      const currentIndex = index + 1;
+      const totalItems = this.props.children.length;
+      const activeOption = this.props.intl.formatMessage({ id: 'Terra.dropdownButton.activeOption' }, { currentItemLabel, currentIndex, totalItems });
+      let ariaLabel = null;
+      if (SharedUtil.isMac()) {
+        if (currentIndex === 1 && totalItems) {
+          ariaLabel = `${this.expanded}${activeOption}`;
```

Comment:
nit:
```suggestion
          ariaLabel = `${this.expanded}, ${activeOption}`;
```
Lets remove the special characters like `dot` and `comma` from translations and include it as part of strings.

Reply:
Done.